src/redux/books/books.js

```diff
@@ -0,0 +1,51 @@
+import { uuid4 } from 'uuid4';
+
```

Comment:
[OPTIONAL] When I feel my input and create a new book it's working. But, if I try to create a new book again the input field is filled again with the previous words. I suggest you remove them when we submit to create a new book.